tasks/banana.js

```diff
@@ -156,6 +158,20 @@ module.exports = function ( grunt ) {
 				} );
 			}
 
+			if ( options.requireKeyPrefix.length ) {
+				if ( typeof options.requireKeyPrefix === 'string' ) {
+					options.requireKeyPrefix = [ options.requireKeyPrefix ];
+				}
+				sourceMessageWrongPrefix = sourceMessageKeys.filter( function ( key ) {
+					return options.requireKeyPrefix
+						.map( function ( prefix ) {
+							return !key.startsWith( prefix );
```

Comment:
(MDN says String#startsWith is Node 4+, and https://node.green/ agrees. We require Node 6. All good.)